localstack/services/ssm/ssm_listener.py

```diff
@@ -19,12 +19,44 @@ def normalize_name(param_name):
 
 def get_secrets_information(name, resource_name):
     client = aws_stack.connect_to_service('secretsmanager')
-    secret_info = client.get_secret_value(SecretId=resource_name)
-    del secret_info['ResponseMetadata']
-
-    result = {'Parameter': {'SourceResult': secret_info, 'Name': name, 'Value':
-              secret_info.get('SecretString'), 'Type': 'SecureString',
+    try:
+        secret_info = client.get_secret_value(SecretId=resource_name)
+        del secret_info['ResponseMetadata']
+        result = {'Parameter': {'SourceResult': secret_info, 'Name': name, 'Value':
+                secret_info.get('SecretString'), 'Type': 'SecureString',
                             'LastModifiedDate': secret_info.get('CreatedDate')}}
+        return result
+    except client.exceptions.ResourceNotFoundException:
+        return None
+
+
+def has_secrets(names):
```

Comment:
nitpick: perhaps a bit more "pythonic":
```
def has_secrets(names):
    for name in names:
        if name.startswith('/aws/reference/secretsmanager'):
            return True
```
(not need to return False - by default `None` is returned which is a falsy value in Python.)